src/gui/search/searchtab.cpp

```diff
@@ -41,31 +41,30 @@
 
 #include "base/utils/misc.h"
 #include "base/preferences.h"
+#include "guiiconprovider.h"
 #include "searchsortmodel.h"
 #include "searchlistdelegate.h"
 #include "searchwidget.h"
 #include "searchtab.h"
+#include "ui_searchtab.h"
 
 SearchTab::SearchTab(SearchWidget *parent)
     : QWidget(parent)
+    , m_ui(new Ui::SearchTab())
     , m_parent(parent)
 {
-    m_box = new QVBoxLayout(this);
-    m_resultsLbl = new QLabel(this);
-    m_resultsBrowser = new QTreeView(this);
+    m_ui->setupUi(this);
+    m_ui->retranslateUi(this);
```

Comment:
>Those are standard methods

`setupUi()` - yes. But not `retranslateUi()`. `retranslateUi()` called internally from `setupUi()`.